site/reference/fn/eval/README.md

```diff
@@ -32,7 +32,16 @@ kpt fn eval [DIR|-] [flags] [-- fn-args]
 DIR|-:
   Path to the local directory containing resources. Defaults to the current
   working directory. Using '-' as the directory path will cause `eval` to
-  read resources from `stdin` and write the output to `stdout`.
+  read resources from `stdin` and write the output to `stdout`. When resources are
+  read from `stdin`, they must be in one of the following input formats:
+
+  1. Multi object YAML where resources are separated by `---`.
+
+  2. [Function Specification] wire format where resources are wrapped in an object
```

Comment:
nit: you're linking twice

Reply:
Fixed. Removed the linking here because links are not being rendered in this section (may be because it's a code block)